### PR TITLE
links css edited

### PR DESCRIPTION
**Pull request by git-devansh**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/960"></a>

### CODE REVIEW

**Review by vikasganiga05 — CHANGES_REQUESTED** [review]
Please remove the `package-lock.json` files from the commit.

**Review by emmalearnscode — CHANGES_REQUESTED** [review]
Thanks for working on this. Just a couple of things:
- remove the package-lock.json from this commit
- remove your profile from this commit and make a separate pull request with just that
- line 23 in links.js is missing the md:w-7 class, please add that in.
If you need any help, please let us know :)

File: src/Components/Links.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ function Links({ links }) {
 
   return (
     <section className="flex justify-content-center mb-4">
-      <div className="flex flex-column w-70">
+      <div className="flex flex-column sm:w-9">
```
Comment (emmalearnscode) [inline src/Components/Links.js:23]:
```suggestion
      <div className="flex flex-column sm:w-9 md:w-7">
```

**Comment by git-devansh** [comment]
Sorry, I'm new to GitHub. I don't know how can I remove files from commit. Any help would be good. Thanks

**Comment by emmalearnscode** [comment]
> Sorry, I'm new to GitHub. I don't know how can I remove files from commit. Any help would be good. Thanks

I think because yours has just updated the version numbers on lines 3 and 9, if you change them back to what they were and commit the changes the file will remove itself from the PR as there will no longer be any differences.

File: package-lock.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "linkfree",
-  "version": "0.58.1",
+  "version": "0.59.0",
```
Comment (emmalearnscode) [inline package-lock.json:3]:
```suggestion
  "version": "0.58.1",
```

File: package-lock.json [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "linkfree",
-      "version": "0.58.1",
+      "version": "0.59.0",
```
Comment (emmalearnscode) [inline package-lock.json:9]:
```suggestion
      "version": "0.58.1",
```

**Review by emmalearnscode — CHANGES_REQUESTED** [review]

**Comment by emmalearnscode** [comment]
@git-devansh how are you getting on with this?

**Comment by emmalearnscode** [comment]
Hi @git-devansh since there are still unresolved conflicts in this file I’m going to close this pull request. Please feel free to make new separate pull requests for your profile and the css issue if you want to and don’t hesitate to reach out for help either here or in discord if you’re unsure of how to do something.

**Comment by git-devansh** [comment]
Thank you I will try to do it

**Pull request closed without merging**